js/lib/alert-bar.js

```diff
@@ -0,0 +1,30 @@
+var $ = require('jquery')
+var callData = require('./communityCall.json')
+
+function shouldShowBanner (date, startTimeString, startDay) {
+  var now = date.getUTCHours()
+  var startTime = parseInt(startTimeString, 10)
+  var dayOfWeek = date.getUTCDay()
+
+  console.log(now, startTime, dayOfWeek)
```

Comment:
It's good form to remove all debug logging from code before we ship it. Please delete this line.